### DISCUSSION TITLE
R.nvim: how can I disable whatever is mapped to "`" while in insert mode?

### DISCUSSION THREAD

**Discussion by weberam2** [LazyVim/LazyVim#4552] [General]
Using the R.nvim extra, when I type '`' while in insert mode, it creates an R code chunk.

Which is useful, but I'd rather remap that to something else.

Can anyone help me disable this? How do I find out what other keys are mapped in insert mode?

Cheers

**Comment by dpetka2001** [comment]
If it's in `insert` mode, then it might come from `cmp-r` source that offers auto-completions for R language. Better read the docs or ask a question at their repo if you can disable it.

**Reply by weberam2** [reply]
Thanks

I tried looking through the docs but maybe I'm too ignorant. I'll ask a question on their repo.

**Reply by weberam2** [reply]
I think I found where it's coming from.
It's in [R.nvim](https://github.com/R-nvim/R.nvim/blob/e3fc31112fac08f451c06a627d8876ab8ce6a393/lua/r/rmd.lua#L13)

How can I disable this?

**Comment by weberam2** [comment]
OK!
I know [where it happens](https://github.com/R-nvim/R.nvim/blob/e3fc31112fac08f451c06a627d8876ab8ce6a393/doc/R.nvim.txt#L1018)

It says to add:
```
lua
   rmdchunk = 0
```
to the config.

and I am so proud I figured out how to make this work:

In my plugins folder, I made a R.lua file:

```
return {
  "R-nvim/R.nvim",
  opts = {
    rmdchunk = 0,
  },
}
```

Hurray!

Thanks everyone
